### PR TITLE
Index has new content

### PR DESCRIPTION
**Pull request by ayojackay**
### Reference
Please reference Issue #1 for details

### Changes
- Added the following:
  - nav
  - two columns
  - footer

_preview_
Screen Shot 2020-09-21 at 7 25 22 PM

### CODE REVIEW

**Review by Pardeepsanghera — COMMENTED** [review]
you're still missing 2 things

**Comment by ayojackay** [comment]
> you're still missing 2 things

@Pardeepsanghera Oh snaps, you're right. Taking care of it now. Sorry!

**Comment by ayojackay** [comment]
@Pardeepsanghera Hey fixed those changes that you requested. Please check it out. Thank you!

File: index.html [hunk 1]
```diff
@@ -1,22 +1,83 @@
...
+    </nav>
+    <!-- (.container-sm>.row.height_80>(.col.color_purple>p{I'm left})+(.col.color_grey>p{I'm right}))*1 -->
+    <div class="container-sm">
+      <div class="row height_80">
+        <div class="col color_purple">
+          <div class="row">
+            <div class="col">
+              <span>p: (559) 800 - 9009</span>
+            </div>
+          </div>
```
Comment (ayojackay) [inline index.html:44]: Hey, this is nice but maybe next time we could use a `p` tag to do the blocking instead of bootstrap in this case.

**Review by Pardeepsanghera — APPROVED** [review]

**Pull request merged**